src/fcntl.rs

```diff
@@ -1080,13 +1080,13 @@ pub fn splice(
 /// # See Also
 /// *[`tee`](https://man7.org/linux/man-pages/man2/tee.2.html)
 #[cfg(linux_android)]
-pub fn tee(
-    fd_in: RawFd,
-    fd_out: RawFd,
+pub fn tee<F: AsFd>(
```

Comment:
For this syscall, even though it requests the 2 fd arguments are all pipes, but technically they still can be different Rust types, so maybe we should use 2 generic types here? Thoughts?

Reply:
makes sense yes.